buzzmobile/constants.yaml

```diff
@@ -1,5 +1,9 @@
 image_width: 1200
 image_height: 1200
 pixels_per_m: 200
```

Comment:
I might define what exactly 'pixels_per_m' and image_width and height are, because they aren't (for example) related at all to the camera we are using, but instead to the projection and clarifying that for future us might be good